### DISCUSSION TITLE
Ubuntu pro/esm

### DISCUSSION THREAD

**Discussion by SimplyCorbett** [Joshua-Riek/ubuntu-rockchip#912] [General]
Will enabling Ubuntu pro/esm cause any issues with the installation?

**Comment by KylinDemons** [comment]
> Will enabling Ubuntu pro/esm cause any issues with the installation?

nop, ubuntu pro works fine for me both in 2204 and 2404

**Reply by SimplyCorbett** [reply]
Thanks!

**Reply by nrpetonr** [reply]
same here, all good
